.github/workflows/rosa-cluster-auto-provision-on-schedule.yml

```diff
@@ -29,6 +29,32 @@ jobs:
     uses: ./.github/workflows/rosa-scaling-benchmark.yml
     with:
       clusterName: gh-keycloak-a # ${{ env.CLUSTER_PREFIX }}-a -- unfortunately 'env.' doesn't work here ${{ env.CLUSTER_PREFIX }}-a
-      skipCreateDeployment: true
-      skipDeleteProject: true
+    secrets: inherit
+
+  cluster-undeploy:
+    needs: run-scaling-benchmark
+    name: ROSA Scheduled Create cluster
+    if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
+    uses: ./.github/workflows/rosa-multi-az-cluster-undeploy.yml
+    with:
+      clusterPrefix: gh-keycloak # ${{ env.CLUSTER_PREFIX }} -- unfortunately 'env.' doesn't work here
+      skipAuroraDeletion: true
+    secrets: inherit
+
+  cluster-deploy-with-persistent-sessions:
+    needs: cluster-undeploy
+    name: ROSA Scheduled Create cluster with Persistent Sessions
+    if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak-benchmark'
+    uses: ./.github/workflows/rosa-multi-az-cluster-create.yml
+    with:
+      clusterPrefix: gh-keycloak # ${{ env.CLUSTER_PREFIX }} -- unfortunately 'env.' doesn't work here
+      enablePersistentSessions: true
+      createCluster: false
+    secrets: inherit
+
+  run-scaling-benchmark-with-peristent-sessions:
```

Comment:
Would it make sense to run functional tests before this step? It can be a follow-up as I am not sure the tests will pass for persistent-sessions.

Reply:
I can run it from my local and then enable it. good idea to run it after the persistent session based deploy.